lib/view/add_label_dialog.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+/// AddLabelDialog:
+///   - dialog for adding a label to a list (used by `ListSettingsDialog`)
+class AddLabelDialog extends StatefulWidget {
+  final Iterable<String> labels;
+
+  const AddLabelDialog(
+      {super.key, required this.labels});
+
+  @override
+  State<AddLabelDialog> createState() => _AddLabelDialogState();
+}
+
+class _AddLabelDialogState extends State<AddLabelDialog> {
+  String searchQuery = '';
+  late TextEditingController textEditingController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return SimpleDialog(
+      children: [
+        ListTile(
```

Comment:
Why is the `TextField` wrapped in a `ListTile`?

Reply:
I was trying to get the `TextField` lined up with the `ListTile`s containing the labels from other lists. I should not have done that this way. Will replace the `ListTile` with `Padding`.